src/test/java/com/google/cloud/spanner/pgadapter/python/django/DjangoBasicTest.java

```diff
@@ -230,4 +262,92 @@ public void testInsertAllTypesAllNull() throws IOException, InterruptedException
     assertEquals(1, mockSpanner.countRequestsOfType(ExecuteSqlRequest.class));
     assertEquals(sqlInsert, mockSpanner.getRequestsOfType(ExecuteSqlRequest.class).get(0).getSql());
```

Comment:
nit: could we add a verification that the statement is executed in auto-commit mode. That is:
1. The execute request includes a BeginTransaction option with a read/write-option=true.
2. There is a commit request on the server.

The reason for the above is that Django is a very big exception to most ORMs. Most ORMs will not use auto-commit by default, but will use read/write transactions that must be committed explicitly by default. We should verify this behavior in order to ensure that our implementation is correct, and to be able to detect potential regressions in the future if we or the Django maintainers make any changes.

Reply:
done